Google Scholar.js

```diff
@@ -282,11 +306,11 @@ var scrapeCase = function (doc, url) {
 	// Citelet is identified by
 	// id="gsl_reference"
 	var refFrag = doc.evaluate('//div[@id="gsl_reference"] | //div[@id="gs_reference"]',
-					doc, null, XPathResult.ANY_TYPE, null).iterateNext();
+		doc, null, XPathResult.ANY_TYPE, null).iterateNext();
 	if (refFrag) {
 		// citelet looks kind of like this
 		// Powell v. McCormack, 395 US 486 - Supreme Court 1969
-		var item = new Zotero.Item("case");
+		// var item = new Zotero.Item("case");
```

Comment:
This I'd want to test again -- I don't *think* we need it but let's double check (GS had me locked out by the time I tried

Reply:
It doesn't look like `item` is ever used, so we can just go ahead and delete that line entirely.